cmd/node.go

```diff
@@ -34,7 +34,7 @@ var nodeCmd = &cobra.Command{
 		}
 
 		filename := loadConfig(check.NODE, bv)
-		runChecks(check.NODE, filename)
+		runChecks(check.NODE, filename, false)
```

Comment:
Same comment as for master, if the user expicitly tries to run the node test cases we should make an attempt